browser/flagr-ui/src/components/Flag.vue

```diff
@@ -257,6 +257,36 @@
                       @save="putFlag(flag)"
                     ></markdown-editor>
                   </el-row>
+                  <el-row style="margin: 10px;">
+                    <h5>
+                      <span style="margin-right: 10px;">Tags</span>
+                    </h5>
+                  </el-row>
+                  <el-row>
+                    <div class="tags-container-inner">
+                      <el-tag v-for="tag in flag.tags"
+                        :key="tag.id"
+                        closable
+                        :type="warning"
+                        @close="deleteTag(tag)">
+                        {{tag.value}}
+                      </el-tag>
+                      <el-autocomplete
+                        class="tag-key-input"
+                        v-if="tagInputVisible"
+                        v-model="newTag.value"
+                        ref="saveTagInput"
+                        size="mini"
+                        :trigger-on-focus="false"
```

Comment:
I think if we set trigger-on-focus to true, it helps to standardize tags creation and reduce typos

Reply:
@zhouzhuojie this was set to false because focus was triggering a save operation which we didn't was causing the saveTagInput method such that it ignores values that are empty but would still cancel your operation instead. This would also conflict with entry process we're using with the save on enter and cancel.